crates/frontend/src/app.rs

```diff
@@ -237,6 +238,14 @@ pub fn app(app_envs: Envs) -> impl IntoView {
                                 }
                             />
 
+                            <Route
+                                ssr=SsrMode::Async
+                                path="/:tenant/workspace"
```

Comment:
Have to do this till the tenant middleware is fixed
```suggestion
                                path="/workspace"
```

Reply:
Okay that works, we can also add this to exclusion list too